spec/factories/activity.rb

```diff
@@ -1,5 +1,5 @@
 FactoryBot.define do
-  factory :activity do
+  factory :__activity do
```

Comment:
I'm surprised you didn't have to declare `class: "Activity"` here, so it knows what to create (unless there's some magic happening under the hood).